source/process/training.rst

```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
 Partners team
 ---------------------------------------------------------
 This is an invite only team meant for Mattermost staff and partners.  This is a team for staff to interact with partners and for partners to interact with other partners. You should prefer public channels when you want partner-to-partner interactions and private channels when you want staff-to-partner interactions.  Remember that any partner added to the team will be able to join any public channel.
+
+---------------------------------------------------------
+Creating New Jira Bug Tickets
+---------------------------------------------------------
+
+A bug is any “obvious error”. If you’re not reporting an obvious error, please file a Story ticket instead. Errors on `unsupported platforms <https://docs.mattermost.com/install/requirements.html>`_ are not considered bugs.
+
+1. Confirm you’re filing a new issue
+---------------------------------------------------------
+
+ - Search existing tickets in Jira to confirm your issue isn’t already filed by someone else.
+
+2. Confirm the bug is not a security issue
+---------------------------------------------------------
+
+ - If your issue involves security, please mark the bug ticket as internal.
+
+3. Information needed on Mattermost bug tickets
+---------------------------------------------------------
+
+ - **Labels:** Add a ``customer-bug`` label if the ticket is based on a customer or a community bug report.
+ - **Environment:** Mattermost server and version, OS and version, Mattermost mobile app version, Mattermost desktop app version (include the ones that are relevant).
```

Comment:
Should we add to Environment (or in a separate item) something about any notable Mattermost configurations (such as HA, Elasticsearch, image proxy, SSO)—anything that might be relevant in helping narrow down the issue?